components/Layout/Images/BlurhashImage.vue

```diff
@@ -54,93 +66,132 @@ export default Vue.extend({
     type: {
       type: String as () => ImageType,
       default: ImageType.Primary
+    },
+    lazy: {
+      type: Boolean,
+      default: () => {
+        return true;
+      }
     }
   },
   data() {
     return {
-      image: ''
+      image: '',
+      loading: true
     };
   },
   mounted(): void {
     const img = this.$refs.img as HTMLElement;
     // We don't pass the item itself as we already did all the tags checking in this component,
     // so doing it again in the mixin is useless.
     this.image = this.getImageUrlForElement(this.type, {
-      itemId: this.item.Id,
+      itemId: this.item.Id as string,
       element: img
     });
   },
   methods: {
+    isPerson(obj: BaseItemDto | BaseItemPerson): obj is BaseItemPerson {
+      if ((obj as BaseItemPerson).Role) {
+        return true;
+      }
+      return false;
+    },
     isValidTag(): boolean {
-      if (
-        (this.item?.ImageTags &&
-          Object.prototype.hasOwnProperty.call(
-            this.item?.ImageTags,
-            this.type
-          )) ||
-        (this.type === ImageType.Backdrop &&
-          this.item?.BackdropImageTags &&
-          this.item?.BackdropImageTags.length > 0)
+      if (!this.isPerson(this.item)) {
+        if (
+          (this.item.ImageTags &&
+            Object.prototype.hasOwnProperty.call(
+              this.item?.ImageTags,
+              this.type
+            )) ||
+          (this.type === ImageType.Backdrop && this.item?.BackdropImageTags)
+        ) {
+          return true;
+        } else {
+          this.onError();
+        }
+      } else if (
+        this.item?.PrimaryImageTag &&
+        this.type === ImageType.Primary
       ) {
         return true;
       } else {
         this.onError();
-        return false;
       }
+      return false;
     },
     canBeBlurhashed(): boolean {
-      if (
-        excludedTypes.includes(this.type) ||
-        (this.item?.ImageBlurHashes as Array<never>).length === 0
-      ) {
+      if (excludedTypes.includes(this.type) || !this.item?.ImageBlurHashes) {
         return false;
+      } else if (!this.isPerson(this.item)) {
+        if (
+          this.type === ImageType.Backdrop &&
+          !this.item?.BackdropImageTags &&
+          this.item.ImageBlurHashes?.Backdrop &&
+          Object.prototype.hasOwnProperty.call(
+            this.item.ImageBlurHashes?.Backdrop as Record<string, string>,
+            (this.item.BackdropImageTags as Array<string>)[0]
+          )
+        ) {
+          return true;
+        } else if (
+          Object.prototype.hasOwnProperty.call(
+            this.item?.ImageBlurHashes,
+            this.type
+          )
+        ) {
+          return true;
+        }
       } else if (
```

Comment:
````suggestion
        } else if (this.item?.ImageBlurHashes?.Primary) {
          return true;
        }
      } else if (
```
To my knowledge, there is a only a primary image for a person?